packages/flutter/lib/src/widgets/heroes.dart

```diff
@@ -692,7 +702,7 @@ class HeroController extends NavigatorObserver {
       return;
     }
 
-    final Rect navigatorRect = _globalBoundingBoxFor(navigator.context);
+    final Rect navigatorRect = _boundingBoxFor(navigator.context, null);
```

Comment:
nit: omit the `null` since it's an optional parameter and the default value for it?